pkg/controller/collection/collection_controller.go

```diff
@@ -145,11 +145,20 @@ func (r *ReconcileCollection) ReconcileCollection(c *kabanerov1alpha1.Collection
 			}
 
 			_collection, err := SearchCollection(collectionName, index)
-			r_log.Error(err, "Could not search the provided index")
+			if err != nil {
+				r_log.Error(err, "Could not search the provided index")
+			}
 			if _collection != nil {
 				collection = _collection
 			}
 		}
+                if collection != nil {
```

Comment:
I don't understand if having multiple repositories should cause more than one collection to be created.  However this does fix the bug that was reported so lets merge this now, and we can investigate the repositories later.